eng/tools/publish-to-npm.ps1

```diff
@@ -8,7 +8,8 @@ param (
   $filterArg="",
   $basicDeployment=$false,
   $devopsFeed=$false,
-  $skipDiff=$false
+  $skipDiff=$false,
+  $pipelineWorkspace
```

Comment:
I would suggest this be the exact path and call it packagesToPublishPath or something along that long. We should avoid creating more sub-directories under here in this script.